docs/contributor/governance.md

```diff
@@ -60,38 +85,45 @@ Below is the list of linters irrelevant for Huskies team.
 | `varcheck`         | superseded by `unused`                                               |
 | `wastedassign`     | superseded by `inefassign`                                           |
 
-## Nolint
-Some linters produce false-positive warnings, so there is a way to suppress them. Consider disabling the linter if linting warnings are routinely ignored or adding to development noise.
+</details>
+
+
+### Nolint
 
-### Inline //nolint
-To suppress a linting warning for a particular line of code, use nolint instruction `//no-lint:{LINTER} // {COMMENT}.` _LINTER_ and _COMMENT_ are two mandatory placeholders with the linter to be suppressed and a reason for the suppression.
+If linting warnings add to the development noise or are routinely ignored (for example, because some linters produce false-positive warnings), consider disabling the linter.
```

Comment:
```suggestion
If linting warnings add noise to the development or are routinely ignored (for example, because some linters produce false-positive warnings), consider disabling the linter.
```